### PR TITLE
intake alliance color selection utilizing screen_selector

### PR DESCRIPTION
**Pull request by Jay9971**
Added to screen/selector:
   get_auton_type - returns the auton type as a short (0 = Blue, 1 = Red, -1 = Skills)

Added to subsystems/intake:
   set_alliance - sets private field alliance_color based on auton_type input. (Blue = false, Red = true, Skills = no change)

Added to main:
  passes auton_type from screen selector into the intake subsystem inside of initialize function

### CODE REVIEW

**Comment by absozero** [comment]
Considering that this code is relevant to the code from the intake, would it make more sense to have the logic in the intake files?

This would remove somewhat irrelevant code from the selector, and put it with the intake, where the color sensor is managed, making more sense in terms of state.

**Comment by Jay9971** [comment]
Pushed changes:
screen/selector.cpp and selector.h are no longer involved.
get_auton_type was added to auton.h and auton.cpp.
Changes to main: auton_type is now passed into the intake subsystem inside of autonomous function.

**Comment by Jay9971** [comment]
Removed get_auton_type from auton.h and auton.cpp.
Removed parameter passing in for intake.set_alliance() in main.cpp
Gave intake header file access to auton header file.

File: src/screen/selector.cpp
Comment (Centei) [inline src/screen/selector.cpp:1]: @Jay9971 You may have not meant to modify this file.

**Review by Centei — APPROVED** [review]
@Jay9971 Everything looks good content wise, however I would check the selector.cpp file as it was not in the scope of your changes.

**Pull request closed without merging**